### ISSUE TITLE
Problem installing betas of 2026.2 on Snapdragon

### ISSUE THREAD

**Issue by andrew-l-d** [nvaccess/nvda#20506]
### Brief summary

When trying to update to beta 6 and beta 7 of NVDA 2026.2 installation fails.  Retrying is ultimately successful.

[failed-install-log.txt](https://github.com/user-attachments/files/30067863/failed-install-log.txt)

[speech-viewer-ad.txt](https://github.com/user-attachments/files/30067876/speech-viewer-ad.txt)

### Steps to reproduce

1. Run nvda_2026.2beta7.exe
2. Accept terms and conditions
3. Select Install
4. Press alt+y

### Actual behavior

The first time in happened a temporary directory opened and the installation beeps continued until I cancelled.  Subsequently the temporary directory opens and the instlalation fails immediately.  With subsequent attempts after restarting and installation completes.

### Expected behavior

The installation process should commence with beeps for several seconds and the install completes.

### NVDA logs, crash dumps and other attachments

Copy of the log file and a copy of Speech Viewer output are attached.

### NVDA type

installed copy

### NVDA version

Updating from NVDA 2026.2 beta 5 to beta 6 and from beta 6 to beta 7. No problems updating from 2026.1.1 to beta 5.

### Have you tried any other versions of NVDA? If so, please report their behaviors.

Not had a problem with previous updates

### Windows version

Windows 11 version 10.0.26200 Build 26200 

### Name and version of other software in use when reproducing the issue

Nil

### Other information about your system

On MS Surface running Snapdragon

### Does the issue still occur after restarting your computer?

Restarting my computer fixed the issue

### If NVDA add-ons are disabled, is your problem still occurring?

I have restarted NVDA with add-ons disabled and the issue still occurs

### Does the issue still occur after you run the System Accessibility Repair Tool (COM Registration Fixing Tool in older versions) in NVDA's tools menu?

I have not run the System Accessibility Repair Tool

**Comment by SaschaCowley** [comment]
Welcome @andrew-l-d ,

We are unable to process this issue as it stands.

- Please reproduce this issue and provide a [log file](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps) of the behaviour. Ensure your log level is set to debug in NVDA's Privacy and Security Settings. Downgrading from 2026.2beta7 to a prior 2026.2 beta should be safe, though if you want to be extra sure, you may make a copy of the `%appData%\nvda` directory.


Kind Regards,
NV Access Software Developers

**Comment by andrew-l-d** [comment]
Log file attached with debug enabled.

[nvda-ad.log](https://github.com/user-attachments/files/30070250/nvda-ad.log)

**Comment by SaschaCowley** [comment]
Hi @andrew-l-d,
That log still appears to be at info level.

If you are comfortable using the command line, run the installer with the `--debug-logging` flag (e.g. `nvda_2026.2beta7.exe --debug-logging`)
Otherwise, run the installer normally, and when the license screen appears, open the NVDA menu and navigate to "Preferences" -> "Settings..." -> "Privacy and Security". Set "Logging level" to "debug", and press "OK". Then proceed with the installation.

**Comment by andrew-l-d** [comment]
Not my day.  Double-checked that it was set to debug and the installation ran correctly.  Will leave the issue open for now and see what happens when the next beta is released.

**Comment by andrew-l-d** [comment]
With the release of beta 8 I tried to update.  It failed twice and I am therefore still on beta 7.  Log file and screen viewer files attached.  Debug was definitely enabled this time.  This is getting serious.

[nvda.log](https://github.com/user-attachments/files/30290139/nvda.log)

[speechviewer.txt](https://github.com/user-attachments/files/30290144/speechviewer.txt)

**Comment by andrew-l-d** [comment]
The issue has become serious.  I can no longer get the update to beta 8 to run.  I tried uninstalling NVDA but get an error.  Looks like I am stuck with Beta 7 on this new machine.  What are the consequences of trashing the NVDA folder from Program Files and from Appdata?

Andrew

**Comment by andrew-l-d** [comment]
The failure to uninstall was at least partly my bad.  When restarting without NVDA running I ran the uninstaller from Program Files/NVDA using Narrator.  The uninstall left all or most files in place.
When running the beta 8 installer the installation was successful but with one carry over bug.  Before attempting the uninstall I tried to stop NVDA from speaking at the logon screen but received an error message shown below.  That error still exist:
C:\Program Files\NVDA\nvda_slave.exe
File system error (-1073741819).
OK
When dismissing the ok another message:
This change requires administrator privileges
appears.  I definitely have admin rights, having installed NVDA and other software.
It will be interesting to see whether the next beta installs.  Looks like I will have to try to remove all files and do a fresh install once 2026.2 is released.

Andrew

**Comment by seanbudd** [comment]
Do you have any anti-malware or system restricting software running? e.g. something a corporate environment would make you install?
Can you reproduce this with add-ons fully disabled? e.g. going into the add-on store and disabling all add-ons, as the add-ons were not disabled in the log provided. Restarting NVDA with add-ons temporarily disabled might not be persisting across the install.

**Comment by andrew-l-d** [comment]
Do you have any anti-malware or system restricting software running? e.g. something a corporate environment would make you install?
There should not be.  It is a private machine.  I have connected to work's WIFI, but cannot imagine that would result in anything being installed without advising me.
When the next beta is released I will disable all add-ons if the problem recurs, as you suggested.  If noone else is having the problem, I suspect a one-off glitch.  Having done the (not complete) uninstall and updating to beta 8 successfully, the glitch may have been resolved.  As mentioned in my previous comment, the issue where I cannot change logon screen settings needs fixing and will probably require a clean install.  Should uninstalling remove everything?

Andrew

**Comment by SaschaCowley** [comment]
@andrew-l-d 

> Should uninstalling remove everything?

Uninstalling NVDA should remove all of its program files, but won't remove your user configuration. If you want to remove your NVDA settings, you can delete the `%AppData%\nvda` folder.

Make sure you restart the machine after uninstalling NVDA, as some files may not be deleted until reboot.

**Comment by andrew-l-d** [comment]
Thanks for all feedback on this issue.  I am reasonably confident that I have found what was causing the problems.
When installing or updating software I press alt+y to respond to the "Do you want this software to make changes ..." prompt.  I have now learned that on the Surface machine it is necessary to tab to the Yes button and press Enter.
Armed with that discovery, I did a fresh install of NVDA using beta 9 without removing the NVDA folder from appdata.  Everything is working as it should.  I will close the issue, apologising for not having picked up the quirky behaviour of the machine.  If the problem does recur when the next beta/RC is released I will reopen it.

Andrew
